src/App.js

```diff
@@ -5,6 +5,8 @@ import Categories from './components/categories/categories';
 import Navbar from './components/navbar/Navbar';
 import store from './redux/configureStore';
 
+export const ENDPOINT = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/OWE8y7l6VJxjIC8WGkOP/books';
```

Comment:
- [ ] Kindly, move the `ENDPOINT` to the `books` slice.

Reply:
Done ✅